sdk/storage/AzureStorageBlob/Source/Transport/URLSessionTransferManager.swift

```diff
@@ -507,6 +508,8 @@ internal final class URLSessionTransferManager: NSObject, TransferManager, URLSe
                     destination: destination,
                     options: transfer.downloadOptions
                 )
+                transfer.downloader?.progress = Int(transfer.bytesTransferred)
+                transfer.downloader?.totalSize = Int(transfer.totalBytesToTransfer)
```

Comment:
This was resulting in bizarre progress on app restart because the downloader progress wasn't being updated from the transfer.